plugins/infinispan91/pom.xml

```diff
@@ -16,6 +15,7 @@
 
    <properties>
       <version.infinispan>${infinispan91.infinispan.version}</version.infinispan>
+      <version.narayanajta>5.0.4.Final</version.narayanajta>
```

Comment:
I removed this in https://github.com/radargun/radargun/pull/518, and you can remove this file from the PR